src/editor/Core/ConfigurationDefaults.re

```diff
@@ -43,6 +43,8 @@ let getDefaultConfigString = configName =>
 {
     "bindings": [
         { "key": "<C-P>", "command": "quickOpen.open", "when": [["editorTextFocus"]] },
+        { "key": "<C-V>", "command": "editor.action.clipboardPasteAction", "when": [["editorTextFocus"]] },
```

Comment:
Does this need an additional clause to prevent the breaking of normal mode visual block mode?

Reply:
Good catch @CrossR - indeed we should have a `when` condition for this.

Reply:
I put together an `insertMode` condition, but really we need more fidelity (something to match the VSCode expression language - ands/ors, string conditions, boolean conditions)